.vscode/settings.json

```diff
@@ -6,5 +6,11 @@
     "**/.sfdx": true
   },
   "eslint.nodePath": "c:\\Users\\James\\.vscode\\extensions\\salesforce.salesforcedx-vscode-lwc-46.17.0\\node_modules",
-  "salesforcedx-vscode-core.show-cli-success-msg": false
+  "salesforcedx-vscode-core.show-cli-success-msg": false,
```

Comment:
To add configuration files to version control are usually decided on a case-by-case scenario. Situations in which config files would be added would be project structure (some projects have them), environment variables, etc. In this situation, the file is different for everybody. I'd add it to _.gitignore_